cluster-up/check.sh

```diff
@@ -26,19 +26,24 @@ fi
 
 KVM_ARCH=""
 KVM_NESTED="unknown"
+KVM_HPAGE="unknown"
 if [ -f "/sys/module/kvm_intel/parameters/nested" ]; then
 	KVM_NESTED=$( cat /sys/module/kvm_intel/parameters/nested )
 	KVM_ARCH="intel"
 elif [ -f "/sys/module/kvm_amd/parameters/nested" ]; then
 	KVM_NESTED=$( cat /sys/module/kvm_amd/parameters/nested )
 	KVM_ARCH="amd"
+elif [ -f "/sys/module/kvm/parameters/nested" ]; then
```

Comment:
May be a good idea to better specify what this commit updates in its description considering the amount of changes

Reply:
Yeah `hack/update-kubevirtci.sh` doesn't really provide any context that I could dump into the commit tbh. Let me add another commit that dumps the actual tag used and I'll rerun the update.